Pipfile

```diff
@@ -26,6 +26,7 @@ tox = ">=3.24"
 types-toml = ">=0.10.0"
 autopep8 = "*"
 mongomock = ">=4.1.2"
+pyarrow = "*"
```

Comment:
Just want to make sure, without `pyarrow` or `fastparquet`, the user can not use ParquetDatanode, right?
If so, it should be a required package.

Reply:
That's right. Since this is quite an optional feature, I think we needn't make it a required package. Pandas also has it as an optional package. Perhaps we should have a discussion on this, if it hasn't already been had.

Reply:
I'm not sure, adding another required package might raise some problems.
@jrobinAV Any thought on this? `pyarrow` is a small package, so I think it should be fine.

Reply:
The Default Pandas engine is `pyarrow`, maybe we can stick to `pyarrow` as well, and add it as a required package.
What would be the purpose of using `fastparquet`?

Reply:
I think we could choose between these 3 approaches:

1. Allow users to specify the `engine` through the Pandas Parquet interface (`pd.DataFrame.to_parquet` and `pd.read_parquet`). Don't add either package as a required package, and preserve the Pandas error if either package isn't installed.
2. Add `pyarrow` (the Pandas default) as a required package. Continue using the Pandas Parquet interface.
3. Add `pyarrow` as a required package. Implement a new wrapper using `pyarrow` for reading/writing Parquet files; i.e., don't use the Pandas interface.

I think the third option will be considerably more work, and it seems a shame to do so when the Pandas interface is already robust and convenient. The second would differ from the first only in that `pyarrow` is added as a required package. We could also maintain the `engine` parameter, which would allow the user to optionally install `fastparquet` and pass `engine="fastparquet"` if they so desired.

What do you think?